### PR TITLE
PMM-10008 Move pmm-agent and pmm-admin to pmm repo

### PR DESCRIPTION
**Pull request by artemgavrilov**
https://jira.percona.com/browse/PMM-10008

- [x] https://github.com/percona/pmm/pull/905
- [x] https://github.com/Percona-Lab/jenkins-pipelines/pull/1566

### CODE REVIEW

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2542-2926151
client docker - perconalab/pmm-client-fb:PR-2542-2926151
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2542-2926151.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2542-2926151&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2542-2926151.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/10162/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/16342/

File: .gitmodules [hunk 1]
```diff
@@ -1,19 +1,11 @@
 [submodule "pmm"]
 	path = sources/pmm/src/github.com/percona/pmm
 	url = https://github.com/percona/pmm
-	branch = main
+	branch = PMM-10008-merge-agnet-and-admin
```
Comment (artemgavrilov) [inline .gitmodules]: TODO: Switch back to main before merging

**Review by nikita-b — APPROVED** [review]
LGTM BUT:
https://github.com/Percona-Lab/jenkins-pipelines/blob/master/pmm/pmm2-prepare-release-candidate.groovy#L45

**Comment by artemgavrilov** [comment]
> LGTM BUT: https://github.com/Percona-Lab/jenkins-pipelines/blob/master/pmm/pmm2-prepare-release-candidate.groovy#L45

Thanks! Fixed: https://github.com/Percona-Lab/jenkins-pipelines/pull/1566

File: build/bin/build-client-binary [hunk 2]
```diff
@@ -14,9 +14,10 @@ extract_source_tarball() {
 
 gobuild_component() {
     local component=$1
-    local extract_path=${2:-"github.com/percona/$component"}
-    local component_path=$3
-    local result_file=${4:-bin/$(basename ${component_path:-$extract_path})}
+    local parent_component=${2:-$component}
```
Comment (BupycHuk) [inline build/bin/build-client-binary:17]: looks like we could do it without having one more variable.

**Review by BupycHuk — APPROVED** [review]

File: build/bin/build-client-binary (hunk 2 above)
Comment (artemgavrilov) [reply build/bin/build-client-binary:17]: It's used in multiple places later

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2542-29c0ad9
client docker - perconalab/pmm-client-fb:PR-2542-29c0ad9
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2542-29c0ad9.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2542-29c0ad9&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2542-29c0ad9.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2542-e12b2ef
client docker - perconalab/pmm-client-fb:PR-2542-e12b2ef
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2542-e12b2ef.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2542-e12b2ef&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2542-e12b2ef.tar.gz

**Comment by JNKPercona** [comment]
pmm2-client testsuite has failed, Please check: BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/9703/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/16431/

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/10181/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/16432/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2542-bcef615
client docker - perconalab/pmm-client-fb:PR-2542-bcef615
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2542-bcef615.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2542-bcef615&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2542-bcef615.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/10185/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/16452/

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2542-c430f45
client docker - perconalab/pmm-client-fb:PR-2542-c430f45
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2542-c430f45.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2542-c430f45&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2542-c430f45.tar.gz

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/10188/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/16463/

**Pull request merged**